src/main/java/org/osiam/ResourceServer.java

```diff
@@ -67,13 +55,16 @@
     @Value("${org.osiam.resource-server.db.vendor}")
     private String databaseVendor;
 
-    @Bean
-    public static PropertySourcesPlaceholderConfigurer properties() {
-        PropertySourcesPlaceholderConfigurer propertySources = new PropertySourcesPlaceholderConfigurer();
-        Resource[] resources = new ClassPathResource[] { new ClassPathResource("resource-server.properties") };
-        propertySources.setLocations(resources);
-        propertySources.setIgnoreUnresolvablePlaceholders(true);
-        return propertySources;
+    public static void main(String[] args) {
+        System.setProperty("spring.jpa.hibernate.naming_strategy", "org.hibernate.cfg.ImprovedNamingStrategy");
+        SpringApplication.run(ResourceServer.class, args);
+    }
+
+    @Override
+    protected SpringApplicationBuilder configure(SpringApplicationBuilder application) {
+        return application
+                .showBanner(true)
+                .sources(ResourceServer.class);
```

Comment:
Yes, according to the [Spring Boot docs](http://docs.spring.io/spring-boot/docs/current/reference/htmlsingle/#howto-create-a-deployable-war-file), we need this. This is necessary if we like to deploy the war into a servlet container. But showBanner set to true is redundant, because it's the default value.